lib/plaid.ex

```diff
@@ -80,6 +80,20 @@ defmodule Plaid do
     make_request_with_cred(method, endpoint, %{}, body, headers, options)
   end
 
+  @doc """
+  Makes request with credentials and options.
+  """
+  @spec make_request_with_cred_and_options(atom, String.t(), map, map, Keyword.t()) ::
+          {:ok, HTTPoison.Response.t()} | {:error, HTTPoison.Error.t()}
+  def make_request_with_cred_and_options(method, endpoint, config, body \\ %{}, httpoison_options \\ []) do
+    request_endpoint = "#{get_root_uri(config)}#{endpoint}"
+    cred = Map.delete(config, :root_uri)
+    request_body = Map.merge(body, cred) |> Poison.encode!()
+    request_headers = get_request_headers() |> Map.to_list()
+    options = httpoison_options
+    request(method, request_endpoint, request_body, request_headers, options)
+  end
+
```

Comment:
hmm why can't we use `make_request_with_cred` and specify the additional options that we have?

Reply:
I tested this approach today but it would produce duplicate keys with ++ options.

Reply:
would update this line to 15 seconds just work without all of these changes?  https://github.com/brexhq/credit_card/blob/main/apps/financials_server/config/prod.exs#L49

Reply:
Yes, but I am not sure if we want to update the timeout for all the Plaid endpoints.

Reply:
gotcha makes sense!